CalibCalorimetry/EcalLaserSorting/interface/LaserSorter.h

```diff
@@ -15,7 +15,7 @@
 #include <vector>
 
 #include "FWCore/Framework/interface/Frameworkfwd.h"
-#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include <FWCore/Framework/interface/stream/EDAnalyzer.h>
```

Comment:
`EDAnalyzers` are typically `global` or `one` since they usually need to see all events.

Reply:
When trying to change to `global`, encounter the same issue as arrived at here: 

https://github.com/atishelmanch/cmssw/commit/f383597277d5a32aded48caf5248a15999ae53bf#r61033990

Reply:
If global is not possible you can make it a one module.

Reply:
Indeed - changing to `one`.